search-books/src/test/java/nix/education/data/TestTagData.java

```diff
@@ -0,0 +1,15 @@
+package nix.education.data;
+
+import nix.education.entity.Tag;
+
+import static nix.education.data.TestConstants.TEST_TAG;
+
+public class TestTagData {
+
+    public static Tag getTag() {
+        Tag tag = new Tag();
+        tag.setTag(TEST_TAG);
+        tag.setId(1L);
+        return tag;
+    }
```

Comment:
Same refactoring as for a book. We don't need to do this initialization each time we're requesting the object

Reply:
Refactored.